fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -2284,6 +2287,19 @@ export class FwsCore {
 
     this.trueNorthRef.set(SimVar.GetSimVarValue('L:A32NX_PUSH_TRUE_REF', 'number'));
 
+    /* V1 callout */
+    const v1 = SimVar.GetSimVarValue('L:AIRLINER_V1_SPEED', 'knots');
+    const v1Threshold = v1 - 4;
+    this.v1SpeedConfirmNode.write(
+      v1 &&
+        (this.adr1Cas.get().valueOr(0) > v1Threshold ||
+          this.adr2Cas.get().valueOr(0) > v1Threshold ||
+          this.adr3Cas.get().valueOr(0) > v1Threshold),
+      deltaTime,
+    );
+    const v1CalloutCondition = this.fwcFlightPhase.get() === 4 && this.v1SpeedConfirmNode.read();
+    SimVar.SetSimVarValue('L:A32NX_AUDIO_V1_CALLOUT', SimVarValueType.Bool, v1CalloutCondition);
```

Comment:
Ideally, we wouldn't set this every iteration of the update loop and only once when the condition is met.

Reply:
Do you mean update only when it changes or only update when the condition is met? The first would mean I have to call GetSimVar for every cycle (might be cheaper), the latter means it doesn't reset, which could be problematic for RTO or turn-arounds

Reply:
Use a subject to store the state, and `subject.sub((v) => SimVar.SetSimVarValue('L:A32NX_AUDIO_V1_CALLOUT', SimVarValueType.Bool, v), true);`

Reply:
good point thanks, just pushed another commit converting it into a subject